src/main/java/core/basesyntax/MyNode.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class MyNode<T> {
```

Comment:
The `MyNode` class should be an inner class within `MyLinkedList` as per the checklist: 'Make `Node<T>` an inner class.'